### PR TITLE
Pull request for the Form section

### PR DESCRIPTION
**Pull request by prius2055**
This is a pull request for the form section of the portfolio project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -222,6 +222,66 @@ <h2>Skills</h2>
...
           </div>
         </section>
+        <section class="form_section">
+          <p>
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </p>
+          <form
+            class="form_element"
+            action="https://formspree.io/f/{form_id}"
```
Comment (didierganthier) [inline index.html:232]: Well done on this form 📄 it only lacks something to work perfectly 😬  **the right ID**, the right ID would be something like `mdovgypd` (this is a fake id) but you can get a real one by creating an account on [Formspree](https://formspree.io/) and creating a new Project, then a new form. Your `form` with my fake ID looks like this:
```
<form class="form_element" action="https://formspree.io/f/mdovgypd">
    <!--Form content-->
</form>
```
action value should be similar to `https://formspree.io/f/{form_id}` not equal to it, I hope this helps 😉

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi Team,

🎉 Good job so far! You're making great progress on this project, but there are still a few areas that need attention before we can move on to the next step.



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Required Changes 🛠
- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

File: index.html (hunk 1 above)
Comment (prius2055) [reply index.html]: Thank you, I have added the form_id

**Review by topeogunleye — APPROVED** [review]
### Hi @prius2055, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
 
## [Optional] suggestions:
* Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**